tpgtools/go.mod

```diff
@@ -3,47 +3,50 @@ module github.com/GoogleCloudPlatform/magic-modules/tpgtools
 go 1.18
```

Comment:
Can we limit changes to this file to only changing the `github.com/GoogleCloudPlatform/declarative-resource-client-library` dependency?

Other than that this looks great, thanks for adding a test!

Reply:
Done. Reverted dependencies other than dcl.